scoreboard/rest.py

```diff
@@ -441,11 +442,12 @@ def get(self):
     def post(self):
         data = flask.request.get_json()
         unlocked = data.get('unlocked', False)
+        normal = utils.normalize_input(data['answer'])
```

Comment:
normal is a bad variable name: it's not clear *what* is normal.  Just 'answer' would be fine here (and elsewhere in this file)